app/src/main/java/ml/docilealligator/infinityforreddit/fragments/HistoryPostFragment.java

```diff
@@ -650,15 +650,9 @@ private int getNColumns(Resources resources) {
     }
 
     private void initializeAndBindPostViewModel(String accessToken) {
-        if (postType == HistoryPostPagingSource.TYPE_READ_POSTS) {
```

Comment:
This is to prepare other types of history posts. So no need to simplify this part.

Reply:
Got it. Reverted.